src/cmd_line.rs

```diff
@@ -123,14 +123,18 @@ pub fn handle_command_line_arguments() {
             .values_of("files")
             .map(|opt| opt.map(|v| v.to_owned()).collect())
             .unwrap_or_default(),
-        maximized: matches.is_present("maximized"),
+        maximized: matches.is_present("maximized")
+            || std::env::var("NeovideMaximized").is_ok()
```

Comment:
nit: I don't think we should have the camel case versions for this. The command line argument change made a good point that we shouldn't use camel case for environment variables.

Could you remove those? Other than that, this is a great change!